src/app/games/martian-chess/tests/MartianChessRules.spec.ts

```diff
@@ -678,12 +676,12 @@ describe('MartianChessRules', () => {
                                                                        0,
                                                                        MGPOptional.empty(),
                                                                        MartianChessRules.STARTING_COUNT_DOWN);
-                spyOn(ErrorLoggerService, 'logError').and.callFake(ErrorLoggerServiceMock.logError);
+                spyOn(Utils, 'logError').and.callFake(ErrorLoggerServiceMock.logError);
 
                 // When calling the clock once more
                 const move: MartianChessMove = MartianChessMove.from(new Coord(1, 7), new Coord(0, 6), true).get();
 
-                // Then the move should throw, cause dev should not do that
+                // Then the move should throw, because the component should not allow it
```

Comment:
non non, c'était mieux avant, revert cette ligne :p !

Reply:
non ! Ça devrait pas être autorisé par le composant, c'est ça qu'on teste, on teste pas que le dev doit pas faire quelque chose

Reply:
then the move should throw, as it is not an expected move